lib/altapay/altapay-php-sdk/Readme.md

```diff
@@ -0,0 +1,45 @@
+Altapay - PHP SDK
```

Comment:
Please use a dependency manager (Composer, Bash or Ant) to load this instead of including it direclty in the repo.

Reply:
For now, we can leave this as it is because the SDK is currently not rebranded from Valitor to Altapay. Since we planned to decommission SDK so we should not consider making this change for now. The library inside this extension was rebranded but the code present in other repository is not currently rebranded.

Reply:
The latest version of the SDK (3.0.0) is rebranded to AltaPay. Could you check if it is compatible or would cause issues.